### ISSUE TITLE
Submitting a new defense (preprint)

### ISSUE THREAD

**Issue by YupingLin171** [robust-ml/robust-ml.github.io#11]
**Name**: {Bandlimiting Neural Networks Against Adversarial Attacks}

**Authors**: {Yuping Lin, Kasra Ahmadi K. A., Hui Jiang}

**Paper**: {https://arxiv.org/abs/1905.12797}

**Code**: {https://github.com/YupingLin171/PostAvgDefense}

**Venue**: {N/A}

**Does the code implement the robust-ml API and include pre-trained models**: {yes}

**Dataset**: {ImageNet, CIFAR-10}

**Threat model**: {ℓ∞(ϵ=8/255)}

**Natural accuracy**: {77.32% on ImageNet, 92.55% on CIFAR-10}

**Claims**: {76.06% on ImageNet, 88.41% on CIFAR-10}

**Note**: {These are the new results and haven't updated to the paper in arXiv yet. The code is up to date. We have tested the robust-ml API implementation on CIFAR-10 but haven't tested on ImageNet (didn't find the "val.txt" file the data provider needs). We just assumed data provider provides data in (N, H, W, C) arrangement.}

**Comment by anishathalye** [comment]
Thank you for your submission! We've added it here: https://www.robust-ml.org/preprints/
